multiaccounts/database.go

```diff
@@ -450,9 +451,29 @@ func (db *Database) publishOnIdentityImageSubscriptions(change *IdentityImageSub
 	}
 }
 
-func (db *Database) DeleteIdentityImage(keyUID string) error {
+func (db *Database) TriggerImageDelete(keyUID string) error {
+	if !db.markedImageForDeletion {
+		return nil
+	}
 	_, err := db.db.Exec(`DELETE FROM identity_images WHERE key_uid = ?`, keyUID)
-	return err
+
+	if err != nil {
+		return err
+	}
+
+	db.markedImageForDeletion = false
+
+	return nil
+}
+
+func (db *Database) DeleteIdentityImage(keyUID string) error {
```

Comment:
I had a discussion with @osmaczko, (thank you! 🙏🏾) and here is the summary:

Instead of introducing a flag and transmitting the images, we can effectively manage the absence of images within the `ContactChatIdentity` in the `messenger_handler`. This approach allows the recipient of the contact code to be informed about the change (deletion of the profile) and update the view accordingly.

Regarding the current code:

At present, we do not address the absence of images because our handling is focused solely on image updates. Enhancing the code to accommodate missing images should effectively address the reported issue. Additionally, adopting this approach will streamline the code, eliminating the need for managing a flag as proposed in this pull request.